frontend/demo/src/api/airyConfig.ts

```diff
@@ -0,0 +1,64 @@
+import { getAuthToken } from "./webStore";
+
+export class AiryConfig {
+  static API_URL = "http://192.168.50.5";
+  static DEVICE_TYPE = "web";
+  static NODE_ENV = process.env.NODE_ENV;
+  static TOP_DOMAIN = "";
```

Comment:
what does this one do? (honest question. I'm not sure I get it)

Reply:
We needed this on the platform to realize shared credentials between inbox and admin.

Reply:
This is the domain where the cookie will be stored if it is not a development environment. @lucapette

Reply:
but we don't need, I removed it